### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by MauriceHendrix**
Setting up github actions

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: build
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.5, 3.6, 3.7, 3.8]
```
Comment (jonc125) [inline .github/workflows/pytest.yml:8]: Why is 3.8 repeated?

File: .github/workflows/pytest.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install .[test];
+    - name: Lint with flake8
+      run: |
+        python -m flake8
+    - name: Test sorting wioth isort
```
Comment (jonc125) [inline .github/workflows/pytest.yml:24]:
```suggestion
    - name: Check imports are ordered correctly
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/ModellingWebLab/cellmlmanip.svg?branch=develop)](https://travis-ci.com/ModellingWebLab/cellmlmanip) [![Documentation Status](https://readthedocs.org/projects/cellmlmanip/badge/?version=latest)](https://cellmlmanip.readthedocs.io/en/latest/?badge=latest) [![Coverage Status](https://codecov.io/gh/ModellingWebLab/cellmlmanip/branch/develop/graph/badge.svg)](https://codecov.io/gh/ModellingWebLab/cellmlmanip/)
+![Build status](https://github.com/ModellingWebLab/chaste-codegen//workflows/build/badge.svg) [![Documentation Status](https://readthedocs.org/projects/cellmlmanip/badge/?version=latest)](https://cellmlmanip.readthedocs.io/en/latest/?badge=latest) [![Coverage Status](https://codecov.io/gh/ModellingWebLab/cellmlmanip/branch/develop/graph/badge.svg)](https://codecov.io/gh/ModellingWebLab/cellmlmanip/)
```
Comment (jonc125) [inline README.md:1]:
```suggestion
![Build status](https://github.com/ModellingWebLab/cellmlmanip/workflows/build/badge.svg) [![Documentation Status](https://readthedocs.org/projects/cellmlmanip/badge/?version=latest)](https://cellmlmanip.readthedocs.io/en/latest/?badge=latest) [![Coverage Status](https://codecov.io/gh/ModellingWebLab/cellmlmanip/branch/develop/graph/badge.svg)](https://codecov.io/gh/ModellingWebLab/cellmlmanip/)
```

**Review by jonc125 — APPROVED** [review]
This looks promising! A few minor comments.

**Comment by jonc125** [comment]
It might be that pint needs upgrading to work on python 3.8?

**Comment by MauriceHendrix** [comment]
> It might be that pint needs upgrading to work on python 3.8?

actually this is a problem: for python3.8 it neds to be updated to Pint 0.10, however Pint 0.10 drops python 3.5 compatability!

**Comment by jonc125** [comment]
> > It might be that pint needs upgrading to work on python 3.8?
> 
> actually this is a problem: for python3.8 it neds to be updated to Pint 0.10, however Pint 0.10 drops python 3.5 compatability!

You can vary which is installed depending on the Python version: https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (MauriceHendrix) [reply .github/workflows/pytest.yml]: mistake!

**Pull request merged**